packages/rest-api-client/src/client/types/plugin/index.ts

```diff
@@ -0,0 +1,26 @@
+type Plugin = {
+  id: string;
+  name: string;
+  isMarketPlugin: boolean;
+  revision: string;
```

Comment:
MUST: ここ`revision`じゃなくて`version`ですね！！！

```suggestion
  version: string;
```

Reply:
すみません、全然気づけていませんでした :bow: 
suggested changeを取り込みました